.gitignore

```diff
@@ -1,3 +1,27 @@
-node_modules
-*indexer_data*
+### macOS ###
+# General
+.DS_Store

```

Comment:
`.DS_Store` is committed into this PR.
Also, `.github/workflows/actions.yml` the ci file should remain.

Reply:
Okay, I'll fix it.